### PR TITLE
Rename environment variables

### LINKED ISSUE #373: Rename environment variables to include `Meilisearch` in it
Currently, we have two environment variables that are being used:

- `HOST`
- `API_KEY`

We would like to rename them and include `Meilisearch` in it to avoid confusion. 

They should be renamed as follow:

- `MEILISEARCH_HOST`
- ~~MEILISEARCH_MASTER_KEY~~ -> `MEILISEARCH_API_KEY`

Make sure to rename them everywhere (code, Readme, tests, environment files, ...)

⚠ Once it's done, a maintainer must update the credentials on Vercel

### PR DESCRIPTION
**Pull request by Smartmind12**
# Pull Request

## Related issue
Fixes #373 

## What does this PR do?
- Rename environment variables to include Meilisearch in it !

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue, or have you listed the changes applied in the PR description (and why they are needed)?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

**Comment by mdubus** [comment]
Hello @Smartmind12 👋 

Thanks a lot for your PR! Can you update your branch to be up-to-date with main? I see conflicts inside the Readme that must be resolved. Thanks! 🙏

**Comment by Smartmind12** [comment]
> Hello @Smartmind12 👋
> 
> Thanks a lot for your PR! Can you update your branch to be up-to-date with main? I see conflicts inside the Readme that should be resolved. Thanks! 🙏

@mdubus I have done the needful kindly review!

**Comment by mdubus** [comment]
Hello @Smartmind12  👋 

I'm really sorry I just realized I made a mistake in the issue, I would like the environment variable to be named `MEILISEARCH_API_KEY` instead of `MEILISEARCH_MASTER_KEY` 🤦‍♀️ 

Would you be kind enough to make the changes? I deeply apologize for this 🙈

**Comment by Smartmind12** [comment]
> Hello @Smartmind12 👋
> 
> I'm really sorry I just realized I made a mistake in the issue, I would like the environment variable to be named `MEILISEARCH_API_KEY` instead of `MEILISEARCH_MASTER_KEY` 🤦‍♀️
> 
> Would you be kind enough to make the changes? I deeply apologize for this 🙈

Haha 😂 No worries at all… will do the changes!

**Comment by mdubus** [comment]
Hello @Smartmind12 👋

The tests are failing because they can't retrieve the host and the api_key. You'll have to rename the `CYPRESS_HOST` into `CYPRESS_MEILISEARCH_HOST`  and `CYPRESS_API_KEY` into `CYPRESS_MEILISEARCH_API_KEY` inside the `.github/workflows/tests.yml` file.

And don't worry about the other PR fixing the same issue, I'll merge yours (the first having been opened) in priority 😉

Thanks a lot and have a good day! ✨

**Comment by Smartmind12** [comment]
> Hello @Smartmind12 👋
> 
> The tests are failing because they can't retrieve the host and the api_key. You'll have to rename the `CYPRESS_HOST` into `CYPRESS_MEILISEARCH_HOST` and `CYPRESS_API_KEY` into `CYPRESS_MEILISEARCH_API_KEY` inside the `.github/workflows/tests.yml` file.
> 
> And don't worry about the other PR fixing the same issue, I'll merge yours (the first having been opened) in priority 😉
> 
> Thanks a lot and have a good day! ✨

Thanks alot!!🙈
I have made those changes...Kindly review!

**Comment by mdubus** [comment]
Thanks a lot for the changes! It works now 🥳 
I cannot merge due to conflicts, can you update your branch? Thanks 🙏

**Comment by Smartmind12** [comment]
> Thanks a lot for the changes! It works now 🥳 I cannot merge due to conflicts, can you update your branch? Thanks 🙏

I checked and found that there are no conflicts and my branch is up-to-date!

**Comment by mdubus** [comment]
> I checked and found that there are no conflicts and my branch is up-to-date!

That's weird because GitHub tells me that there are conflicts: 

Capture d’écran 2022-10-18 à 09 55 31

I just tried to go to your branch and found that there is a conflict inside the Readme: 

Capture d’écran 2022-10-18 à 10 08 21

Would you please try it once again? Thanks a lot 🙏

**Comment by Smartmind12** [comment]
Rename environment variables by Smartmind12 · Pul…

That is wierd because i am unable to update branch since its already updated and shows no conflicts.😬

**Comment by Smartmind12** [comment]
@mdubus i have updated the readme again...Kindly check if it works now!

File: README.md [hunk 1]
```diff
@@ -124,8 +133,8 @@ serve .next
 
 To connect to your own Meilisearch instance, two environment variables should be provided:
 
-- `HOST`: the URL of your Meilisearch instance
-- `API_KEY`: an API key with at least search rights
+- `HOST`: the URL of your Meilisearch instance.
+- `API_KEY`: an API key with at least search rights.
```
Comment (mdubus) [inline README.md:137]: Can you update this with `MEILISEARCH_HOST` and `MEILISEARCH_API_KEY` ? 
You can also remove the dots at the end 😉

Comment (Smartmind12) [reply README.md:137]: I had added the full stop to commit changes in the readme😂 cool i will make the changes.

Comment (Smartmind12) [reply README.md]: I have done the changes 

> Can you update this with `MEILISEARCH_HOST` and `MEILISEARCH_API_KEY` ? You can also remove the dots at the end 😉

I have made the changes!

**Comment by mdubus** [comment]
bors try

**Comment by alallema** [comment]
bors try

**Comment by mdubus** [comment]
Thanks a lot for your PR @Smartmind12 ✨ 🦕 

GitHub was having some issues (I still saw the `merge conflict` problem, but my colleague didn't 🤔), so we merged it on a temporary branch and I'll merge it after. 

Thanks again for your help and your patience! Have a nice day! ☀️

**Pull request merged**